roles/seqreports/defaults/main.yml

```diff
@@ -9,15 +9,19 @@ checkqc_bcl2fastq_output_path: Unaligned
 seqreports_data_path: "{{ sw_path }}/seqreports"
 
 seqreports_venv_name: seqreports-env
-seqreports_user: seqreports  # Only used in dev and local environment
-seqreports_user_uid: 506  # Only used in dev and local environment
-seqreports_group: 'MOLM-GG-BioInfo'  # Only used in dev and local environment
-seqreports_group_uid: 504
 
+nextflow_version: 21.04.1
```

Comment:
right now, I think it would be best to use the Nextflow version from the Nextflow role since that is the one that is deployed and available. For that reason, I would suggest to drop this variable here.

Reply:
See my comment below.